src/BaGet.Database.MySql/Migrations/20210919191554_RemoveReleaseNotesMaxLength.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BaGet.Database.MySql.Migrations
+{
+    public partial class RemoveReleaseNotesMaxLength : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<DateTime>(
+                name: "RowVersion",
+                table: "Packages",
+                rowVersion: true,
+                nullable: true,
+                oldClrType: typeof(DateTime),
+                oldType: "timestamp(6)",
+                oldNullable: true)
+                .Annotation("MySql:ValueGenerationStrategy", MySqlValueGenerationStrategy.ComputedColumn);
```

Comment:
This seems sketch but from my testing it doesn't seem to cause any problems... 😅